statistics_msgs/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>statistics_msgs</name>
```

Comment:
Maybe `topic_statistics_msgs`?

Is there a plan to add services in the future? Then `_interfaces` would be the more future proof suffix.

Reply:
There is no plan to add services right now. 

These messages are also used for system metrics messages in https://github.com/ros-tooling/system_metrics_collector so I didn't want to specifically tie them to topic statistics. And since the message represents min, max, stddev etc. it seemed appropriate. 

Is the concern here that `statistics_msgs` is too generic?

Reply:
I was going solely by the description of the package in the manifest. If the messages are used for more the name can stay as is. Maybe update the package description then?

Reply:
Updated the description.